src/client/features/search/query-entity-info.js

```diff
@@ -160,6 +139,56 @@ const getUniprotInfo = ids => {
   });
 };
 
+const getHgncInfo = hgncSymbols => {
+  return Promise.all(Object.keys(hgncSymbols).map( hgncSymbol => {
+    return ServerAPI.getHgncInformation( hgncSymbol ).then( result => {
+      const geneResults = result.response.docs;
+      return geneResults.map(gene => {
+
+        let databaseId = {};
+        Object.entries(dbInfos).forEach( ( [k,v] ) => {
```

Comment:
We're missing UniProt and Gene Cards here, both of which are available in the response from `rest.genenames.org/fetch/symbol` as `uniprot_ids`, `symbol`.

Reply:
Is HGNC symbol always the same as GeneCards id?

Reply:
https://www.genecards.org/Guide/AboutGeneCards

> Linking to particular GeneCards
> ...
> To link using an external id, use "https://www.genecards.org/cgi-bin/carddisp.pl?id=YYY" 
(where YYY is a UniProt accession, EntrezGene id, Ensembl id, or HGNC id).
> Optionally, a source for the id can be specified by adding "&id_type=" to the url. The possible types are "uniprot", "entrezgene", "ensembl", and "hgnc". To avoid confusion, it is strongly recommended to use the id_type parameter, especially for numeric ids.

Reply:
It seems that the 'gene symbol' is the hgnc approved symbol in the cases that I tried...

Reply:
[Wikipedia entry](https://en.wikipedia.org/wiki/GeneCards)
> The core gene list is based on approved gene symbols published by the HUGO Gene Nomenclature Committee (HGNC).[7][8]